graphql/chunker.js

```diff
@@ -18,7 +19,19 @@ const getFirstNToken = (text, maxTokenLen) => {
     return text;
 }
 
+const determineTextFormat = (text) => {
+  const htmlTagPattern = /<[^>]*>/g;
```

Comment:
So any text with anything that looks like a tag in it will get the HTML treatment?  Seems aggressive.